rc/tools/lint.kak

```diff
@@ -89,12 +89,14 @@ define-command \
                     '
                         BEGIN { OFS=":"; FS=":" }
 
-                        /:[1-9][0-9]*:[1-9][0-9]*:/ {
+                        /^[^:]+:([1-9][0-9]*:){1,2} / {
                             $1 = ENVIRON["kak_bufname"]
-                            if ( $2 == 1 ) {
+                            $2 += line_offset
+                            if ( $3 ~ /^ / ) {
+                                $2 = $2 FS
```

Comment:
why do we need to add that `:` to $2 ? cant we use this logic (`$3 ~ /^ /`) in the subsequence awk script to detect if we had a column ?